lib/govuk_publishing_components/presenters/meta_tags.rb

```diff
@@ -48,6 +49,8 @@ def add_core_tags(meta_tags)
         primary_publisher = content_item.dig(:links, :primary_publishing_organisation)
         primary_publisher = primary_publisher.first[:title] unless primary_publisher.blank?
         meta_tags["govuk:primary-publishing-organisation"] = primary_publisher unless primary_publisher.blank?
+
+        # Some browse topics are nested in the content item, we want to grab these for GA4 tracking.
```

Comment:
Unrelated to the PR - just added this comment to improve the code for the GA4 meta tags.